indra/llcharacter/lljoint.cpp

```diff
@@ -67,10 +67,11 @@ void LLVector3OverrideMap::showJointVector3Overrides( std::ostringstream& os ) c
 	map_type::const_iterator max_it = std::max_element(m_map.begin(),
 													   m_map.end(),
 													   attachment_map_iter_compare_key<map_type::value_type>);
-	for (const map_type::value_type& pos_pair : m_map)
+	for (map_type::const_iterator it = m_map.begin();
+		 it != m_map.end(); ++it)
 	{
-		const LLVector3& pos = pos_pair.second;
-		os << " " << "[" << pos_pair.first <<": " << pos << "]" << ((pos_pair==(*max_it)) ? "*" : "");
+		const LLVector3& pos = it->second;
+		os << " " << "[" << it->first <<": " << pos << "]" << ((it==max_it) ? "*" : "");
 	}
 }
```

Comment:
Revert the above change.  The previous `for(item : container)` pattern is better than explicit iterators, I think.

This same feedback applies to all the other instances below, modulo specific feedback as necessary.